src/notso_glb/utils/gltfpack.py

```
@@ -0,0 +1,103 @@
+    if texture_compress:
+        cmd.append("-tc")
+
+    if mesh_compress:
+        cmd.append("-cc")
+
+    if simplify_ratio is not None:
+        simplify_ratio = float(simplify_ratio)
+        if not (0.0 <= simplify_ratio <= 1.0):
+            raise ValueError(
+                f"simplify_ratio must be in [0.0, 1.0], got {simplify_ratio}"
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)